.changeset/rich-rivers-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/hydrogen': patch
+---
+
+Adding support for B2B to the customer account client.
```

Comment:
maybe mentioned it is unstable and what those API are. Both cart and customer account client